.github/workflows/posemesh-sdk.yml

```diff
@@ -0,0 +1,101 @@
+name: Posemesh SDK
+on: [push]
```

Comment:
Do we need to build it on every push? Maybe just on main branch?

Reply:
yeah, feel free to change/recommend what should we put there